index.js

```diff
@@ -4,9 +4,11 @@ var chokidar = require('chokidar');
 var xtend = require('xtend');
 
 module.exports = watchify;
-module.exports.args = {
-    cache: {}, packageCache: {}
+module.exports.defaults = function (opts) {
+    return xtend({cache: {}, packageCache: {}}, opts);
 };
+// DEPRECATED - will be removed in a future version
+module.exports.args = watchify.defaults();
```

Comment:
How about using the builtin node.js deprecate method? This will allow the use of `--trace-deprecation` to find the offending code.

```javascript
var util = require('util')

Object.defineProperty(module.exports, 'args', {
  get: util.deprecate(watchify.defaults, 'watchify.args: use watchify.default(...) instead')
})
```

This would also solve the problem of shared cache since each `var _ = watchify.args` would get its own object. Example:

```javascript
var a = watchify.args
var b = watchify.args

console.log('is same:', (a === b).toString())
```